expyfun/visual/_visual.py

```diff
@@ -991,25 +991,10 @@ def __init__(self, ec, file_name, pos=(0, 0), units='norm', scale=1.,
         self._units = units
         self._center = center
         self.set_scale(scale)  # also calls set_pos
-        self.set_visible(visible)
-        self._update_background()
-
-    def _update_background(self):
```

Comment:
This turned out not to be working anyway, and the `GLubyte` operation is sloooow.  Hit all sorts of pyglet errors trying to just change the already existing texture's image data rather than creating a new `ImageData` object, and eventually gave up.